docs/api.rst

```diff
@@ -4210,6 +4211,7 @@ Dealing with trades
    :reqjson int from_timestamp: The timestamp from which to query. Can be missing in which case we query from 0.
    :reqjson int to_timestamp: The timestamp until which to query. Can be missing in which case we query until now.
    :reqjson string location: Optionally filter trades by location. A valid location name has to be provided. If missing location filtering does not happen.
+   :reqjson list[string] ignored_names: If ``ignored_names`` is present then ``location`` should also be provided. The exchanges with the ignored names will not be synced with the DB, but their trade will be returned if they were already synced.
```

Comment:
We said that this was not needed and that the query doesn't change. Is just the behaviour. We already have all the data we need in the database. The bug is that we were not filtering properly